### ISSUE TITLE
[Bug]: First start tour not working

### ISSUE THREAD

**Issue by LennardF1989** [lowcoder-org/lowcoder#237]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On step 3 of creating the app, a wrong URL is used for the user datastore. This seems a blatant find/replace of Openblocks:
https://63621db87521369cd06514c2.mockapi.io/api/lowcoder/users doesn't work, but https://63621db87521369cd06514c2.mockapi.io/api/openblocks/users does.

That said, regardless of a working URL, you are stuck on this step as the next button won't do anything. This was also an issue on Openblocks, by the way.

### Expected Behavior

You can continue after step 3.

### Steps to reproduce

Spin up a new instance, register an account and follow the tour.

**Comment by FalkWolsky** [comment]
Yep. That is true. We think we model a completely new demo case around a small eCommerce solution as we started in the documentation already. That would round it up. Thank you very much for bringing this up!

**Comment by FalkWolsky** [comment]
We fixed this now in v2.2.0
